pkg/cmd/server/api/v1/types.go

```diff
@@ -367,6 +367,15 @@ type OAuthConfig struct {
 	SessionConfig *SessionConfig `json:"sessionConfig"`
 
 	TokenConfig TokenConfig `json:"tokenConfig"`
+
+	// Templates allow you to customize pages like the login page.
+	Templates *OAuthTemplates `json:"templates"`
+}
+
+type OAuthTemplates struct {
+	// Login is a file on the asset server to use as a template for login page customization.
```

Comment:
`Login is a path to a file containing a go template, used to render the login page` (not on the asset server)